helm/aws-load-balancer-controller/README.md

```diff
@@ -108,10 +108,11 @@ The old controller must be uninstalled completely before installing the new vers
 ### Kubectl installation
 If you had installed the previous version via kubectl, uninstall as follows
 ```shell script
-$ kubectl delete deployment -n kube-system alb-ingress-controller
 # Find the version of the current controller
```

Comment:
The version is useful only for the `kubectl delete -f` option.
There are two ways to delete,  
1. `kubectl delete deployment -n kube-system alb-ingress-controller`
2. Find the version, and `kubectl delete -f ...`